### DISCUSSION TITLE
[灌水] 治安局混进绳网？

### DISCUSSION THREAD

**Discussion by Galaxy22223333** [share121/inter-knot#1318] [其他]
绳网难道不是绳匠的聚集地吗，也算半个非法网站了吧，治安局的人是怎么混进来的？

**Comment by Inter-Knot-User** [comment]
搞不懂，现在很害怕

**Comment by Galaxy22223333** [comment]
太恐怖了，据说连法厄同都被抓了

**Comment by lllliufeng** [comment]
还好我根本抢不到委托，不然怕是要进去（

**Comment by su2569** [comment]
太恐怖了

**Comment by kaijinti** [comment]
钓鱼执法

**Comment by ZhuYuan-148** [comment]
要守法哦
![1723330806354](https://github.com/user-attachments/assets/087b76cf-abfe-4bb2-9f34-abd49f10c169)

**Comment by GanlinSama** [comment]
哇，是朱鸢警光！大家快跑口牙！
